node_red/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM nodered/node-red:latest
+MAINTAINER Irwan Fathurrahman<meomancer@gmail.com>
+ADD settings/package.json /data/package.json
+ADD settings/flows.json /data/flows.json
+ADD settings/settings.js /data/settings.js
+ADD settings/patches /data/patches
+
+COPY setup.sh /usr/local/bin/
+RUN ln -s /usr/local/bin/setup.sh
```

Comment:
What is the destination for the symlink_

Reply:
Fixed @timlinux